NitroxPatcher/NitroxPatcher.csproj

```diff
@@ -75,13 +75,15 @@
     <Compile Include="Patches\EscapePod_Awake_Patch.cs" />
     <Compile Include="Patches\Fabricator_OnCraftingBegin_Patch.cs" />
     <Compile Include="Patches\GameSettings_SerializeInputSettings_Patch.cs" />
+    <Compile Include="Patches\Persistent\GameInput_Initialize_Patch.cs" />
+    <Compile Include="Patches\Persistent\GameInput_SetupDefaultKeyboardBindings_Patch.cs" />
+    <Compile Include="Patches\Persistent\uGUI_OptionsPanel_AddTabs_Patch.cs" />
+    <Compile Include="Patches\Persistent\GameSettings_SerializeSettings_Patch.cs" />
     <Compile Include="Patches\ItemsContainer_NotifyAddItem_Patch.cs" />
     <Compile Include="Patches\ItemsContainer_NotifyRemoveItem_Patch.cs" />
     <Compile Include="Patches\MedicalCabinet_OnHandClick_Patch.cs" />
     <Compile Include="Patches\NitroxPatch.cs" />
     <Compile Include="Patches\Openable_PlayOpenAnimation_Patch.cs" />
-    <Compile Include="Patches\Persistent\GameInput_Initialize_Patch.cs" />
```

Comment:
Can't these stay in their original position (and move the other `Persistent` patches here)? I just sorted everything in an earlier PR.